src/interpreter/index.ts

```diff
@@ -184,6 +184,33 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
         }
     }
 
+    callNamedFunction(functionName: string) {
```

Comment:
Can you parking lot how this actually works for us?  Does this just find the first callback function with that exact function name?  What if different components have the same callback name?

Reply:
As synced offline, the `RoSGNode` will iterate over all the observers attached and will call each function by name using the reference to the interpreter.